RevenueCat/Scripts/CustomerInfo.cs

```diff
@@ -87,18 +87,18 @@ public CustomerInfo(JSONNode response)
 
         public override string ToString()
         {
-            return $"{nameof(Entitlements)}: {Entitlements}, " +
-                   $"{nameof(ActiveSubscriptions)}: {ActiveSubscriptions}, " +
-                   $"{nameof(AllPurchasedProductIdentifiers)}: {AllPurchasedProductIdentifiers}, " +
-                   $"{nameof(LatestExpirationDate)}: {LatestExpirationDate}, " +
-                   $"{nameof(FirstSeen)}: {FirstSeen}, " +
-                   $"{nameof(OriginalAppUserId)}: {OriginalAppUserId}, " +
-                   $"{nameof(RequestDate)}: {RequestDate}, " +
-                   $"{nameof(AllExpirationDates)}: {AllExpirationDates}, " +
-                   $"{nameof(AllPurchaseDates)}: {AllPurchaseDates}, " +
-                   $"{nameof(OriginalPurchaseDate)}: {OriginalPurchaseDate}, " +
-                   $"{nameof(ManagementURL)}: {ManagementURL}, " +
-                   $"{nameof(NonSubscriptionTransactions)}: {NonSubscriptionTransactions}, " +
```

Comment:
I updated all the `ToString`s to use line breaks rather than commas, it was super hard to understand their contents otherwise.